setup.cfg

```diff
@@ -0,0 +1,59 @@
+[flake8]
+filename = *.py,*.pyx,*.pxd,*.pxi
+max-line-length = 120
+max-complexity = 10
+ignore = E731,E228,E225,E226,E704,W504
+per-file-ignores = __init__.py:F401
+
+[isort]
+line_length = 120
+no_lines_before = LOCALFOLDER
+lines_after_imports = 2
+length_sort_straight = True
+
+[pydocstyle]
+convention = numpy
+add-ignore = D100,D102,D105,D301
+
+[metadata]
+name = tip
+version = 0.1.0
+license_files = LICENSE.txt
+
+[mypy]
+ignore_missing_imports = True
```

Comment:
As I remember we don't use mypy anymore?

Reply:
IDK about it, stole the config from our most configured project and removed few specific ignores. If we did, I'll remove it.

Reply:
So, if you include these lines into setup.cfg then your code probably passes mypy and pylint checks? 😂

Reply:
Well now it passes!

```
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
Check Yaml...........................................(no files to check)Skipped
Check for added large files..............................................Passed
pylint...................................................................Passed
mypy.....................................................................Passed
flake8...................................................................Passed
isort....................................................................Passed
```